### PR TITLE
Add desription of -noKeyAuth command line option for german translation

### PR DESCRIPTION
**Pull request by mkochenough**
I updated the change, fixed the errors and converted the file to \uXXXX syntax.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @daniel-beck @albers

File: cli/src/main/resources/hudson/cli/client/Messages_de.properties [hunk 1]
```diff
@@ -6,6 +6,7 @@ CLI.Usage=Jenkins Kommandozeilenschnittstelle (Jenkins CLI)\n\
   -i KEY              : Datei mit privatem SSH-Schl�ssel zur Authentisierung\n\
   -p HOST\:PORT       : HTTP-Proxy-Host und -Port f�r HTTPS-Proxy-Tunnel. Siehe http://jenkins-ci.org/https-proxy-tunnel\n\
   -noCertificateCheck : �berspringt die Zertifikatspr�fung bei HTTPS. Bitte mit Vorsicht einsetzen.\n\
+  -noKeyAuth          : �bersrpingt die Authentifizierung mit einem private SSH Schl�ssel. Nicht kombinierbar mit -i\n\
```
Comment (albers) [inline cli/src/main/resources/hudson/cli/client/Messages_de.properties]: There are two typos:
s/Übersrpingt/Überspringt/ and s/private/privaten/

Comment (daniel-beck) [inline cli/src/main/resources/hudson/cli/client/Messages_de.properties]: SSH-Schlüssel mit Bindestrich.

**Comment by daniel-beck** [comment]
Is this the correct file encoding? In the GitHub notification email I just got � for all the umlauts, so I'm wondering how it looks in the application. Maybe encoding them as `\uHHHH` would be better in general?

**Comment by daniel-beck** [comment]
Thanks for the ping @oleg-nenashev

**Pull request merged**